node/service/src/chain_spec/vara.rs

```diff
@@ -702,18 +702,21 @@ fn testnet_genesis(
 
     balances.push((bank_account, EXISTENTIAL_DEPOSIT));
 
-    GenesisConfig {
+    RuntimeGenesisConfig {
         system: SystemConfig {
             // Add Wasm runtime to storage.
             code: wasm_binary.to_vec(),
+            ..Default::default()
```

Comment:
whats the default params?
shouldn't we set them for local node matching real runtime?

Reply:
There is just config.
```rust

		#[serde(skip)]
		pub _config: sp_std::marker::PhantomData<T>,
```

Reply:
> There is just config.
> 
> ```rust
> 		#[serde(skip)]
> 		pub _config: sp_std::marker::PhantomData<T>,
> ```

Yeah, just found this out while was looking on substrate changes between branches. I'd manually specify these configs in order not to miss anything in future that would be added into genesis configs and migrations.